### PR TITLE
DEV-2603 - Fixed bug in proto path.

### PR DESCRIPTION
**Pull request by nomad-software**
Moved the protos into the correct path.

### CODE REVIEW

**Review by mryan321 — COMMENTED** [review]
Oh man - I'm not sure about this now (sorry) 😊 but very much open for thoughts 😄 

In my specs I also dropped the `ennismore...` from package name in proto:
https://github.com/ennismore/em-common/blob/3056b71d06d56f531dd64e743363b460ea485731/api/domain/v1/common.proto#L3

This was mainly from following the pattern here: https://buf.build/buf-examples/observabilitytypes

Also seemed sensible from an import pov too: `domainv1 "go.buf.build/protocolbuffers/go/ennismore/common/domain/v1"`

But now I'm thinking - maybe better to add back the `ennismore.ohip` and `ennismore.domain` dirs 🙈  more like https://buf.build/googleapis/googleapis & https://github.com/googleapis/googleapis/tree/master/google

Wha ya reckon?

**Comment by nomad-software** [comment]
> Wha ya reckon?

Yeah, I think you're right. :smile:

File: buf.gen.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: v1
 managed:
```
Comment (mryan321) [inline buf.gen.yaml:2]: minor one really - i spotted this on demo:

```
plugins:
  - remote: buf.build/protocolbuffers/plugins/go:v1.28.0-1
    out: gen/
    opt: paths=source_relative
  - remote: buf.build/library/plugins/go-grpc:v1.1.0-2
    out: gen/
    opt:
      - paths=source_relative
      - require_unimplemented_servers=false
 ```

remote plugins mean no local deps - will be better in CI too 👍

**Review by mryan321 — APPROVED** [review]
lgtm - one buf.gen.yaml suggestion but not major 😊 good stuff 👍

File: buf.gen.yaml (hunk 1 above)
Comment (nomad-software) [reply buf.gen.yaml:2]: That is awesome!

**Review by db80 — APPROVED** [review]

**Pull request merged**